server/apps/distributed-app/docs/modules/ROOT/partials/IsMarkedAsSpam.adoc

```diff
@@ -14,4 +14,15 @@ As an example, here is a part of a mailet pipeline which can be used in your Loc
         <targetFolderName>Spam</targetFolderName>
     </mailet>
 <!-- End of SpamAssassing mailets pipeline -->
+....
+
+In order to use this with `rspamd`, we need to declare a condition for the matcher.
+Eg: With the recipient header for RSpamD being *org.apache.james.rspamd.status*,
+then the configuration would be:
+
+....
+<!-- SpamAssassing mailets pipeline -->
```

Comment:
```suggestion
<!-- RSpamD mailets pipeline -->
```

You should put the RSpamD mailet prior this. Also you need to explain that one would need to drop the RSpamD jar in `extensions-jars` folder.